js/src/client.ts

```diff
@@ -93,7 +93,7 @@ interface CreateRunParams {
   run_type: string;
   execution_order?: number;
   id?: string;
-  start_time?: number;
+  start_time: number;
```

Comment:
Why not auto-create in the `createRun` call? seems like an arg users shouldn't have to worry about 99% of the time

Reply:
That works too, one issue we see is that the current behavior (ie. created in the server) ends up with runs getting inconsistent start times, because it's not guaranteed the server will handle requests in the order they are sent. Creating it here in the sdk code feels better

Reply:
Yeah I agree. Probably good to add a default in the SDK before putting in the queue